test/e2e/fixture/fixture.go

```diff
@@ -573,6 +593,21 @@ func WithTestData(testdata string) TestOption {
 	}
 }
 
+func runFunctionsInParallel(t *testing.T, functions map[string]func()) {
+	t.Helper()
+
+	var wg sync.WaitGroup
+
+	for _, function := range functions {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			function()
```

Comment:
Are we relying on `os.Exit`s in the parallel functions for error handling?

Reply:
Hm, that's a good point, probably need to return the errors to the main function.

Reply:
It's not that easy. I've checked and it's relying on `os.Exit` after logging `Fatal`. Is that okay?

Reply:
I've figured out a way.